src/components/axiosConfig.ts

```diff
@@ -1,6 +1,18 @@
 import axios from 'axios'
+import {getClientResponse} from '../redux/clientResponse'
+import {store} from '../redux/store'
+
 
 const requestTaskApi = axios.create({
-	baseURL: 'http://localhost:5000'
+	baseURL: 'http://localhost:5000',

```

Comment:
Zapoznaj się z zasadami https://12factor.net/ 
Pomoże to utrzymać repozytorium w porządku.

Co do samego trzymania adresów odnosi się zasada: https://12factor.net/processes